src/util.ts

```diff
@@ -11,3 +15,28 @@ export function getUserFromMention(client: Client, mention: string): User | unde
 	const id = /^<@!?(\d+)>$/.exec(mention)?.[1];
 	return id ? client.users.cache.get(id as Snowflake) : undefined;
 }
+
+function getApplicationCommandFromCommandName(client: Client, commandName: string): ApplicationCommand | undefined {
+	return client.guilds.cache
+		.get(getState().config.guildId)
+		?.commands.cache.filter(cmd => cmd.name === commandName)
+		.first();
+}
+
+export function getMemberCommands(member: GuildMember): Command[] {
+	return getCommands().filter(async command => {
```

Comment:
filter and async don't play well together I don't think, we might wanna use map, promise.all, then filter?